designer/apps/pages/models.py

```diff
@@ -51,11 +54,72 @@ class ProgramPage(Page):
     content_panels = Page.content_panels + [
         FieldPanel('uuid', classname="full"),
         FieldPanel('idp_slug'),
+        InlinePanel('program_homepage', label="Program Homepage", max_num=1),
         InlinePanel('program_documents', label="Program Documents", max_num=1),
         InlinePanel('branding', label="Program Page Branding", max_num=1),
     ]
 
 
+class ProgramHomepage(models.Model):
+    """
+    Section on the Program Page linking back to the Program's Homepage and describing what learners can expect to find
```

Comment:
Yeah actually this came up during the grooming of this ticket - i'm not sure what to call this thing I mostly used "ProgramHomepage" as a place holder because I thought "ManageYourDegree" would be a confusing name for the model.

Reply:
Hmmm. `ExternalProgramWebsite`? Really anything that doesn't have `page` in the name would work fine. #WagtailDownsides

Reply:
Yeah that works for me - I also didn't love overloading the term ```page``` in the wagtail context for reasons